sros2_cmake/cmake/ros2_security_generate_artifacts.cmake

```diff
@@ -12,12 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-macro(ros2_secure_node)
-  # ros2_secure_node(NODES <node_1> <node_2>...<node_n>)
+macro(ros2_security_generate_artifacts)
```

Comment:
Nit: this is a mouthful

How about `sros2_generate_artifacts`? this would convey information about where the macro comes from as well as which sros2 verb it's linked to

Reply:
Sounds good!

Reply:
Done in https://github.com/ros2/sros2/pull/177/commits/2db3d4e8bff1b56e6621b170a78feeafcd4b1fbd.